python/cog.py

```diff
@@ -305,6 +348,28 @@ async def on_component(self, ctx: ComponentContext):
             embed.set_field_at(index=0, name=name, value=jump_url)
             await ctx.edit_origin(embed=embed)
 
+    async def send_files_as_script(self, ctx:SlashContext, files: List[Dict]):
+        """
+        Send a list of Discord files as a script to download them.
+
+        Args:
+            ctx: The originating context.
+            files: The files to send to the context.
+        """
+
+        script_top = export_template[0:EXP_TEMPLATE_SPLIT_IDX]
+        dict_strs = list(map(lambda d: str(d) + ',', files))
+        script_bottom = export_template[EXP_TEMPLATE_SPLIT_IDX:]
+        script_lines = []
+        script_lines.extend(script_top)
+        script_lines.extend(dict_strs)
+        script_lines.extend(script_bottom)
+        script = '\n'.join(script_lines)
+
+        f = io.BytesIO(bytes(script, 'utf-8'))
+        await ctx.send(file=discord.File(f, filename="export.py"))
```

Comment:
Add a message explaining how to run the script - how to install python, which version, and how to run the script.

Reply:
This is the message for now:

```
# This script will download all the files you requested.
# To run it, you will need Python 3 installed on your computer.
# For Windows, you can install it from the MS store, or from
# https://www.python.org/downloads/windows/
# For macOS, you can install it from https://www.python.org/downloads/macos/
# For Linux, use your distor's package manager.
# If you need help running the script, see
# https://realpython.com/run-python-scripts/#how-to-run-python-scripts-using-the-command-line
```

If you really want I can write out instructions for running Python scripts from the terminal for each OS.